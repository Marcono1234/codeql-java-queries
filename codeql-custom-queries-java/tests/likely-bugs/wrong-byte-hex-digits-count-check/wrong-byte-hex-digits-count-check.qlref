./likely-bugs/wrong-byte-hex-digits-count-check.ql
