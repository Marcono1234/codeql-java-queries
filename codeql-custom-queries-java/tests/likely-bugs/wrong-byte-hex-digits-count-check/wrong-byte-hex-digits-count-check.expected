| Test.java:4:13:4:18 | ... < ... | Might incorrectly check hex digits count of signed byte value |
| Test.java:5:13:5:18 | ... > ... | Might incorrectly check hex digits count of signed byte value |
| Test.java:6:13:6:19 | ... <= ... | Might incorrectly check hex digits count of signed byte value |
| Test.java:7:13:7:19 | ... >= ... | Might incorrectly check hex digits count of signed byte value |
| Test.java:8:13:8:18 | ... > ... | Might incorrectly check hex digits count of signed byte value |
| Test.java:9:13:9:18 | ... < ... | Might incorrectly check hex digits count of signed byte value |
| Test.java:10:13:10:19 | ... >= ... | Might incorrectly check hex digits count of signed byte value |
| Test.java:11:13:11:19 | ... <= ... | Might incorrectly check hex digits count of signed byte value |
| Test.java:14:13:14:23 | ... > ... | Might incorrectly check hex digits count of signed byte value |
